manifests/preferred.json

```diff
@@ -2082,6 +2082,12 @@
       "docPath": "readable-stream",
       "category": "preferred"
     },
+    {
+      "type": "documented",
```

Comment:
somehow this one came back

any idea how that keeps happening?

Reply:
My bad, fixed it 😅